src/slack_notify.ml

```diff
@@ -66,16 +70,22 @@ let execute token username channel icon_url icon_emoji msg =
   let channel = Slacko.channel_of_string channel in
   let chat = Slacko.Channel channel in
   let msg = Slacko.message_of_string msg in
+  let attachments =
```

Comment:
Pulling in `maybe` from `slacko.ml` and writing a function to do the `[Slacko.attachment ~text:txt ()]` costs about as much space as it saves by not having the pattern match, and the end result is harder to read at a glance. I'll remove the unnecessary parens, though.